core/controllers/topic_editor.py

```diff
@@ -541,14 +544,29 @@ def get(self, topic_id: str) -> None:
         skill_creation_is_allowed = (
             role_services.ACTION_CREATE_NEW_SKILL in self.user.actions)
 
+        curriculum_admin_usernames = (
```

Comment:
admin -> admins, not a serious issue, but "admin_usernames" sounds like one admin might have multiple usernames

Reply:
yes make sense but in other parts like in email_manger.py we just use admin. So should I change it or it's fine?

Reply:
Consistency is a good argument, so let's keep it then, nice observation!